cypress/e2e/HW17.1.cy.js

```diff
@@ -0,0 +1,31 @@
+describe ('Header and Footer Buttons', ()=> {
+    beforeEach(() => {
+        cy.visit('https://guest:welcome2qauto@qauto.forstudy.space/') 
+      });
+
+
+    it ('HeaderButtons', ()=> {
```

Comment:
add informative description to each test. For example
```

    it ('Sign up button should be visible', ()=> {
```

Reply:
good point)
Thanks